submissions/Temu4/memory_pair_game/js/js/script.js

```diff
@@ -0,0 +1,131 @@
+const content = document.getElementById('content');
+const defaultButton = document.getElementById('defaultButton');
+const cards = document.getElementsByClassName('card');
+const gameScore = document.getElementById('gameScore');
+const gameResult = document.getElementById('gameResult');
+
+//arr wirh images (classes)
+const imagesForCards = [
+  'card_poroshenko',
+  'card_timoshenko',
+  'card_grycenko',
+  'card_zelenskyi',
+  'card_liashko',
+  'card_vaider',
+  'card_poroshenko2',
+  'card_timoshenko2',
+  'card_grycenko2',
+  'card_zelenskyi2',
+  'card_liashko2',
+  'card_vaider2'
+];
+
+//shuffle items in array
+const Shuffle = arr => {
+  for (let j, x, i = arr.length; i; j = parseInt(Math.random() * i), x = arr[--i], arr[i] = arr[j], arr[j] = x);
+  return arr;
+};
+
+//process cards content
+const processCardsContent = (arr, content) => {
+  for (let i = 0; i < arr.length; i++) {
```

Comment:
>Please, use more modern approach for array loop.
It's not an array.
I used it, because`forEach()` method don't word with `HTMLCollection`. Fix it in [this]() commit.